package-lock.json

```diff
@@ -16396,6 +16396,7 @@
       "version": "1.1.11",
       "resolved": "https://registry.npmjs.org/brace-expansion/-/brace-expansion-1.1.11.tgz",
       "integrity": "sha512-iCuPHDFgrHX7H2vEI/5xpz07zSHB00TpugqhmYtVmMO6518mCuRMoOYFldEBl0g187ufozdaHgWKcYFb61qGiA==",
+      "dev": true,
```

Comment:
Just to confirm, is this change intentional?

Reply:
Yeah, we should only change `package-lock.json` if one of the `package.json` files changed.  Let's revert this.

Reply:
I ran npm i in the server package to fix a type error in outcomes.ts

```
function getValidationErrorExpression(error: ValidationError): string[] | undefined {
  // ValidationError can be AlternativeValidationError | GroupedAlternativeValidationError | UnknownFieldsError | FieldValidationError
  if (error.type === 'field') {
    return [error.path];
  }
  return undefined;
}
```
The error.type and error.path kept producing errors.